plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/admin/TeleportCommand.java

```diff
@@ -23,8 +23,9 @@ public void execute(Player player, String[] args) {
 
         Guild guild = GuildValidation.requireGuildByTag(args[0]);
 
-        Region region = guild.getRegion();
-        when(region == null || region.getCenter() == null, messages.adminNoRegionFound);
+        when(!guild.hasRegion(), messages.adminNoRegionFound);
+        Region region = guild.getRegion().get();
+        when(region.getCenter() != null, messages.adminNoRegionFound);
```

Comment:
Dlaczego w ogóle center to może być null

Reply:
W teorii nie powinien...